.github/workflows/tests_wf.yml

```diff
@@ -37,11 +41,11 @@ jobs:
     - name: Set env variables
       run: echo "PYTEST_ARGS=--cov-append --cov=pydicom" >> "$GITHUB_ENV"
     - name: Check-out code
-      uses: actions/checkout@v3
```

Comment:
I wonder why the dependabot has not changed this yet...

Reply:
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot#enabling-dependabot-version-updates-for-actions

Looks like needs a section in dependabot.yml.  Might as well add that here...

Reply:
Okay ... added.  Nice catch!

Reply:
Ah, I thought we already had that! That explains it, thanks!